ee/agent/storage/sqlite/migrations/000001_kv.up.sqlite

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS keyvalue_pairs (
+    pair_id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    key_name TEXT NOT NULL UNIQUE,
+    key_value TEXT
```

Comment:
```suggestion
    key TEXT NOT NULL UNIQUE,
    value TEXT
```
would be more traditional

Reply:
I didn't want to do `key` because that's a reserved keyword in sqlite

Reply:
huh. Okay!

Reply:
I also figure that once I rename this table per https://github.com/kolide/launcher/pull/1515#discussion_r1433386798, I can update the column names to something less clunky 🙂

Reply:
Went with `name` and `value`

Reply:
Yeah, `name` is good